src/main/java/utils/Parser.java

```diff
@@ -0,0 +1,58 @@
+package utils;
+
+import commands.AddCommand;
+import commands.Command;
+import commands.DeleteCommand;
+import commands.ListCommand;
```

Comment:
`Parser.java` uses `commands.FindCommand` and requires an import for it.

```suggestion
import commands.FindCommand;
import commands.ListCommand;
```